solr_configs/catalog-production-v2/conf/solrconfig.xml

```diff
@@ -35,6 +35,16 @@
     <updateLog>
       <str name="dir">${solr.core0.data.dir:}</str>
     </updateLog>
+    <!-- Hard commit every 60 minutes -->
+    <autoCommit>
+      <maxDocs>10000</maxDocs>
+      <maxTime>36000000</maxTime>
+      <openSearcher>false</openSearcher>
+    </autoCommit>
+    <!-- Soft commit every 5 minutes -->
```

Comment:
Have we tested the autocommit of every 5 min in other apps? Will it maybe increase the CPU?

Reply:
Most of the other apps are configured this way. I lifted it from DPUL. I think 5 minutes is reasonable; feel free to ask around for other opinions!

Reply:
It should be fine, soft commits are very light and don't do a lot of writes.

Reply:
That's fine. We can always change it back if there is a need to.